renovate.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": [
+    "github>openstack-k8s-operators/renovate-config:default.json5"
+  ],
```

Comment:
https://github.com/openstack-k8s-operators/renovate-config/blob/main/default.json5

this is not a typo.

https://github.com/openstack-k8s-operators/openstack-operator/blob/main/renovate.json#L3

Reply:
Thanks for the explicit clarification.